src/com/mambu/apisdk/services/AccountingService.java

```diff
@@ -125,4 +170,79 @@ public GLAccount getGLAccount(String glCode, String fromDate, String toDate) thr
 
 		return serviceExecutor.execute(getGLJournalEntries, params);
 	}
+
+	// Params definitions for posting GL Journal Entries. See MBU-1737
+	private static String debitAccountParam = "debitAccount";
+	private static String debitAmountParam = "debitAmount";
+	private static String creditAccountParam = "creditAccount";
+	private static String creditAmountParam = "creditAmount";
+
+	/**
+	 * Post GL Journal Entries
+	 * 
+	 * @param entries
+	 *            a list of entries with the GL transaction details. Must not be null. At least one debit and one credit
+	 *            entry must be specified. Any number of journal entries may be posted with a given date and branch id
+	 *            as long as the standard accounting rules apply. For each entry its glCode, entryType and amount must
+	 *            not be null.
+	 * @param branchId
+	 *            a branch id.
+	 * @param date
+	 *            The date of the posting of the journal entry. Must be not null
+	 * @param notes
+	 *            transaction notes
+	 * @return created journal entries
+	 * @throws MambuApiException
+	 */
+	public List<GLJournalEntry> postGLJournalEntries(List<ApiGLJournalEntry> entries, String branchId, String date,
+			String notes) throws MambuApiException {
+		// POST
+		// api/gljournalentries/branchId=b01&date=2012-02-05&debitAccount1=1003&debitAmount1=-0.05&creditAccount1=1003&creditAmount1=-0.05"

```

Comment:
Are the parameters added to the URL or posted in the body? They should be part of the body.